doc/rados/troubleshooting/troubleshooting-pg.rst

```diff
@@ -736,11 +736,16 @@ cluster and that they operate only on local files:
 
 Here, ``--num-rep`` is the number of OSDs that the erasure code CRUSH rule
 needs, ``--rule`` is the value of the ``rule_id`` field that was displayed by
-``ceph osd crush rule dump``. This test will attempt to map one million values
-(in this example, the range defined by ``[--min-x,--max-x]``) and must display
-at least one bad mapping. If this test outputs nothing, all mappings have been
-successful and you can be assured that the problem with your cluster is not
-caused by bad mappings.
+``ceph osd crush rule dump``. This test will simulate a number of PG placements
```

Comment:
My sense is that the leading ``ceph osd crush rule dump`` is superfluous and should be removed.  Thoughts?

Reply:
I think it's useful as a roundabout way of saying "the `rule_id` is the ID of the CRUSH rule" by means of providing a command to get it. It answers both the question of what the rule ID is, as well as how to get it.